### PR TITLE
feat: update workflow with go-server

### PR DESCRIPTION
**Pull request by hetao92**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,29 +1,45 @@
...
 RUN npm install
-COPY . /nebula-graph-studio/
+COPY . /web/
 
 # build and remove front source code
-RUN npm run build && npm run tsc && rm -rf app/assets/*
-COPY ./app/assets/index.html  /nebula-graph-studio/app/assets/
+ENV NODE_OPTIONS=--max_old_space_size=2048
+RUN npm run build
+RUN ls && ls dist
```
Comment (veezhang) [inline Dockerfile]: This is debug command, you can remove it.

File: scripts/rpm/nebula-graph-studio.service [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Nebula Graph Studio
+AssertPathExists=PREFIX_TEMPLATE/nebula-graph-studio/
+[Service]
+Restart=always
+StartLimitInterval=0
+RestartSec=10
+WorkingDirectory=PREFIX_TEMPLATE/nebula-graph-studio/
+ExecStart=PREFIX_TEMPLATE/nebula-graph-studio/nebula-studio-server
```
Comment (nianiaJR) [inline scripts/rpm/nebula-graph-studio.service:9]: nebula-studio-server rename to server? more short & simply

File: scripts/rpm/nebula-graph-studio.service [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+[Install]
+WantedBy=multi-user.target
```
Comment (jerryliang64) [inline scripts/rpm/nebula-graph-studio.service:12]: what's it used for?

**Review by jerryliang64 — DISMISSED** [review]
LGTM

**Review by jerryliang64 — APPROVED** [review]
LGTM

**Pull request merged**